### PR TITLE
[SIS-156] Home page bugs and QOL changes

### PR DESCRIPTION
**Pull request by yuemily24**
# Description

- When there are no new activity cards, the page will display some text “Nothing here!” and the refresh button will say “Check for cards from the past week”
- Fixed axios undefined request
- Undefined variables throughout code

### CODE REVIEW

File: src/services/pdf.js [hunk 1]
```diff
@@ -11,15 +11,15 @@ export const createPDF = async lessonPlan => {
...
   // add title
-  html.push(header(`Lesson Plan: ${lessonPlan['lessonPlanName']}`, 1));
+  html.push(header(`Lesson Plan: ${lessonPlan.lessonPlanName}`, 1));
 
   html.push(moduleInformation(lessonPlan['Warm Up'], 'Warm Up'));
   html.push(moduleInformation(lessonPlan['Main Lesson'], 'Main Lesson'));
   html.push(moduleInformation(lessonPlan['Cool Down'], 'Cool Down'));
   // eslint-disable-next-line no-extra-boolean-cast
-  if (!!lessonPlan['Notes']) {
+  if (!!lessonPlan.Notes) {
```
Comment (yuemily24) [inline src/services/pdf.js:20]: @ramyzhang Does this screw anything up? Linting automatically changed it to this format (from `!!lessonPlan['Notes']` to `!lessonPlan.Notes`) but I can change it back

Comment (ramyzhang) [reply src/services/pdf.js:20]: yap just tested it and it's fine still!! i was just copying baker's syntax tbh

**Review by ramyzhang — APPROVED** [review]
looking good!!!! nice catch w the home bug - the home screen actually still displays the old ACs automatically for me so i think the case only occurs if you install it after there hasn't been new ACs for a while

**Comment by yuemily24** [comment]
> looking good!!!! nice catch w the home bug - the home screen actually still displays the old ACs automatically for me so i think the case only occurs if you install it after there hasn't been new ACs for a while

Yes I checked behaviour and if it doesn't find any new ACs, it won't erase what you currently have

**Pull request merged**